### ISSUE TITLE
General: Optimize video dialogs

### ISSUE THREAD

**Issue by emeijerperfectfit** [PerfectFit-project/testing-tickets#1]
General:
- duration of videos needs to be changed into actual duration in all dialogs
- messages "Bedankt voor het kijken van de video [etc]" and "Typ 1 voor duidelijk en 2 [etc]" should not come immediately but when the user is almost done watching the video
- sentence "Je bent klaar met dit video onderdeel" should come before the next activity is introduced

**Comment by wbaccinelli** [comment]
@emeijerperfectfit a couple of questions:

- the message after the video is sent now 30s after the link is sent, as described in [the excel file](https://teams.microsoft.com/_?tenantId=ca2a7f76-dbd7-4ec0-9108-6b3d524fb7c8#/xlsx/viewer/teamsSdk/https:~2F~2Fleidenuniv1.sharepoint.com~2Fsites~2FTeamAio~2FShared%20Documents~2FGeneral~2FContent%20folder~2FVirtual%20Coach%20Dialogs%20and%20Resources~2F1%20-%20C1%20Dialogs~2FFOR%20SPRINT%20NOV%2030%20---%20C1.12%20Dialog%20watch%20this%20video.xlsx?threadId=19:4eaffeda135540b2ae879fb071d73654@thread.tacv2&subEntityId=%257B%2522viewParams%2522%253A%2522id%253D%25252Fsites%25252FTeamAio%25252FShared%252520Documents%25252FGeneral%25252FContent%252520folder%25252FVirtual%252520Coach%252520Dialogs%252520and%252520Resources%25252F1%252520%25252D%252520C1%252520Dialogs%2526viewid%253Dc0b07892%25252D4a36%25252D4ab0%25252D8d81%25252Df9ac52e34585%2522%257D&baseUrl=https:~2F~2Fleidenuniv1.sharepoint.com~2Fsites~2FTeamAio&fileId=524b49b0-f566-442e-b69b-5afa708b4c9a&ctx=openFilePreview&viewerAction=edit). Is this not happening or should we change the delay to a different amount? In this case, which would this be?
 
- the final sentence is also implemented at the end also as described in the excel file. Indeed, if the videos is in the preparation phase, the sentence is before the next activity. This flow is also optimal for the implementation, so if this is not really a priority, I would not change it. What do you think?

**Comment by aversluis1** [comment]
@wbaccinelli: 
Regarding the first point: the message appears very quickly after the link is send, so it probably is 30 second. Would change the time to the time it takes to watch the video. 

Regarding the second point: the sentence 'Je bent klaar met dit video onderdeel' came - when I tested it - at a very strange point (in the preparation intro video). So first I watched the videos and answered question on whether video was clear or not. Then it said i could continue to the next part (lasting 2 /3 min) and I could answer whether i wanted to do it now or later. AFter this it showed the message "je bent klaar met dit video onderdeel". You would expect this message after answering the question on whether video is clear or not. Anyway, I just tested it again, and now this did not happen (last message was not shown).
